enterprise/frontend/src/embedding-sdk/lib/theme/embedding-color-palette.ts

```diff
@@ -49,6 +49,7 @@ export const SDK_TO_MAIN_APP_COLORS_MAPPING: Record<
   "background-hover": ["bg-light"],
   "background-secondary": ["bg-medium"],
   "background-disabled": ["background-disabled"],
+  "background-inverse": ["bg-black", "background-inverse"],
```

Comment:
Maps `background-inverse` as `bg-black` is used throughout the app, including as the tooltip's default background.